packages/eslint-plugin/tests/rules/no-unsafe-unary-minus.test.ts

```diff
@@ -0,0 +1,31 @@
+import { RuleTester } from '@typescript-eslint/rule-tester';
+
+import rule from '../../src/rules/no-unsafe-unary-minus';
+import { getFixturesRootDir } from '../RuleTester';
+
+const rootDir = getFixturesRootDir();
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2015,
+    tsconfigRootDir: rootDir,
+    project: './tsconfig.json',
+  },
+  parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('no-unsafe-unary-minus', rule, {
+  valid: [
+    '+42;',
+    '-42;',
+    '-42n;',
+    '(a: number) => -a;',
```

Comment:
[Bug] This isn't valid syntax. It might coincidentally work for now, but there's no guarantee it always will.

For this and the other `=>` cases:

```suggestion
    'const _ = (a: number) => -a;',
```

...though, I'd personally suggest using a `declare const` instead, as it's a little more clear what's being tested. 🤷 up to you!

```ts
declare const a: number;
-a;
```

Reply:
> This isn't valid syntax

In what sense is it not? 🤔

Reply:
OH wow I am wrong! That is valid syntax! I guess it makes sense that you could have an arrow function as an expression... never mind! 😄